### PR TITLE
Updates to the guide

### PR DESCRIPTION
**Pull request by gareth-d85**
I have updated some old links and added links to some online guides that are very good.

---

## ✅️ By submitting this PR, I have verified the following:

- [x] Added descriptive name to PR
  - Your PR should NOT be called `Update index.md`, since it does not help the maintainer understand what has been changed.
  - For example, if you create a **Variables** article inside the **Python** directory, the pull request title should be **Python: add Variables article**.
  - Additional PR title examples:
    - **Git: edit Git Commit article**
    - **PHP: create PHP section and add Data Structures article**
- [x] Reviewed necessary formatting guidelines in [`CONTRIBUTING.md`](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md).
- [x] No plagiarized, duplicate, or repetitive content that has been directly copied from another source.

### CODE REVIEW

**Comment by gareth-d85** [comment]
I hope I have done this properly this time!

File: src/pages/bootstrap/index.md [hunk 1]
```diff
@@ -66,4 +65,4 @@ themes that build on the Bootstrap framework to provide a more customized and st
 <a href='http://builtwithbootstrap.com/' target='_blank' rel='nofollow'>Built with Bootstrap</a>: Showcase of sites built using Bootstrap
 <a href='https://github.com/twbs/bootlint' target='_blank' rel='nofollow'>Bootlint</a>: HTML linter for projects using Bootstrap
 <a href='https://bootsnipp.com/' target='_blank' rel='nofollow'>Bootsnipp</a>: Design elements and code snippets for Bootstrap
-<a href='http://expo.getbootstrap.com/resources/' target='_blank' rel='nofollow'>More Resources</a>: Code, theme, and add-on resources for Bootstrap
\ No newline at end of file
+<a href='http://expo.getbootstrap.com/resources/' target='_blank' rel='nofollow'>More Resources</a>: Code, theme, and add-on resources for Bootstrap
```
[the remark this answers is not in the document]
Comment (gareth-d85) [reply src/pages/bootstrap/index.md]: I did not make that change in the pull request

**Comment by gareth-d85** [comment]
I have fixed all the changes I have made

**Comment by Asjas** [comment]
Thank you for your contribution to the page! :+1:
We're happy to accept these changes, and look forward to future contributions. :tada:

**Pull request merged**